application/classes/Ushahidi/Repository/Config.php

```diff
@@ -49,21 +55,49 @@ public function update(Entity $entity)
 
 		$config = \Kohana::$config->load($group);
 
+		// Intercom count datasources
+		if ($group === 'data-provider') {
+			$intercomData['num_data_sources'] = 0;
```

Comment:
Yeah, I'm not sure where else we could intercept the datasources. The ideal would really be to just have an occasional query that gives us stats on deployments. That would work best in the context of the single DB though. Then if that's needed in intercom we could push from our own stats collector.